bad_connector_woocommerce/models/sale_order/common.py

```diff
@@ -51,6 +52,36 @@ class SaleOrder(models.Model):
         readonly=True,
     )
 
+    is_fully_returned = fields.Boolean(
+        string="Fully Returned",
+        compute="_compute_is_fully_returned",
+        store=True,
+        readonly=True,
+    )
+
+    @api.depends("order_line.qty_delivered", "order_line.product_uom_qty")
```

Comment:
This compute method deliveres the wrong value, even when no return is created, we should have to change the logic, and update the boolean value based on the origin_returned_move_id of original moves.